jiracmd/request.go

```diff
@@ -74,8 +74,17 @@ func CmdRequest(o *oreo.Client, globals *jiracli.GlobalOptions, opts *RequestOpt
 	}
 	defer resp.Body.Close()
```

Comment:
This looks good to me, however, there is a chance both body and err can be nil. 

These blocks should actually be: 

If resp.Body == nil  {
  // return error 
}

Then the rest of the code is safe.